src/lib/components/labeledInput/labeledInput.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { componentPropTypes, defaultComponentPropTypes } from '../../utils/componentPropTypes';
+import styles from './labeledInput.module.scss';
+
+const LabeledInput = ({
+  theme,
+  name,
+  value,
+  error,
+  label,
+  className,
+  type,
+  onChangeFunction,
+  handleValidations,
+  helperText,
+  ...inputProps
+}) => {
+  const handleChange = event => {
+    const inputValue = event.target.value;
+    const isWrongValidation = handleValidations(inputValue);
+    onChangeFunction(name, inputValue, isWrongValidation);
```

Comment:
Good to keep doing this as the other input 👍

Reply:
Nice to have: do this in a outsider function so you don't have to repeat the code here and in the other Label

Reply:
Done 👍